### PR TITLE
Yul fuzzer: Introduce basefee() and blobbasefee() builtins

### PR DESCRIPTION
**Pull request by bshastry**
Depends on https://github.com/ethereum/solidity/pull/14755

### CODE REVIEW

**Comment by cameel** [comment]
This looks like it should be based on #14755 for tests to pass. Also, doesn't it depend on #14782 too?

**Review by cameel — DISMISSED** [review]
Looks fine but needs a rebase.

**Comment by bshastry** [comment]
> Looks fine but needs a rebase.

Rebased.

**Review by cameel — DISMISSED** [review]

File: test/tools/ossfuzz/protoToYul.cpp [hunk 1]
```diff
@@ -762,6 +764,24 @@ void ProtoConverter::visit(NullaryOp const& _x)
...
 	}
+	case NullaryOp::BASEFEE:
+		// Replace calls to basefee() on unsupported EVMs with a dictionary
+		// token.
+		if (m_evmVersion.hasBaseFee())
+			m_output << "basefee()";
+		else
+			m_output << dictionaryToken();
+		break;
+	}
```
Comment (cameel) [inline test/tools/ossfuzz/protoToYul.cpp]: Actually, syntax is broken here. This does not build because of the two extra `}`.

Comment (bshastry) [reply test/tools/ossfuzz/protoToYul.cpp]: Sorry, fixed this now.

**Review by nikola-matic — APPROVED** [review]

**Pull request merged**